mycroft/skills/event_scheduler.py

```diff
@@ -54,14 +55,19 @@ class EventScheduler(Thread):
     """
     def __init__(self, bus, schedule_file='schedule.json'):
         super().__init__()
-        data_dir = expanduser(Configuration.get()['data_dir'])
 
         self.events = {}
         self.event_lock = Lock()
 
         self.bus = bus
         self.is_running = True
-        self.schedule_file = join(data_dir, schedule_file)
+        old_path = join(expanduser(Configuration.get()['data_dir']),
+                        schedule_file)
+        if isfile(old_path):
+            self.schedule_file = old_path
```

Comment:
can we `shutil.move` the file to the new directory automatically instead?

Reply:
Please read https://github.com/MycroftAI/mycroft-core/pull/2578#issuecomment-655919861 and following comments, we already discussed if we should just move everything over or keep old locations.

Reply:
just use the combo_lock and it should be safe to move?

Reply:
other possibility is to support both locations for a while, but copy the file to new location, and on load if new location exists, delete from old location, this way you do a 2 step move in an async way that is safe across processes

Reply:
@forslund

Reply:
I think this should be safe to move since it's not really exposed to the user. This should only be used from the skills process anyways so combo-lock shouldn't be necessary